qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -152,9 +152,18 @@
             cmd_read = "cat"
             cmd_change_black = "sed -i 's/guest-file-[a-zA-Z]*,//g' /etc/sysconfig/qemu-ga"
             cmd_verify = "grep -nr 'BLACKLIST_RPC=.*guest-file' /etc/sysconfig/qemu-ga"
+            cmd_del = "rm -rf"
             Windows:
                 file_path = "C:\"
                 cmd_read = "type"
+                cmd_del = "del /f /q"
+            variants:
+                - basic_test:
+                    basic_test = yes
+                - write_with_param:
+                    write_params_test = yes
+                - fsfreeze:
+                    fsfreeze = yes
```

Comment:
That's not a good idea to have so long condition statement for unrelated functions. Two approach for it:
1. To utilize existed `gagent_check_type`, have 3 independent `gagent_check_` functions, and organize their sharing part to 2 functions(one for the part before the try, and another for after finally).
2. Having the three condition statement as 3 independent function, calling them by `getattr(function_name)`.

I will prefer the approach 1. But 2 is also better than current implementation.